nyxl-theme/src/css/_header.scss

```diff
@@ -0,0 +1,420 @@
+#banner {
+	background: $midnightblue-color;
+	max-width: none;
+	width: 100%;
+
+	a {
+		font-weight: normal;
+	}
+
+	input:-webkit-autofill,
+	input:-webkit-autofill:hover,
+	input:-webkit-autofill:focus,
+	input:-webkit-autofill:active,
+	.form-control:focus {
+		box-shadow: 0 0 0 100px $midnightblue-color inset;
+		-webkit-text-fill-color: #FFF;
+	}
+
+	li:focus,
+	li > a:focus ,
+	li > a span:focus ,
+	ul:focus {
+		outline-color: $nyred-color;
+	}
+
+	input {
+		font-family: $nav-font;
+	}
+
+	.portlet-topper {
+		z-index: 1;
```

Comment:
```z-index``` values should match what we have defined in https://github.com/liferay/liferay-portal/blob/7.0.4-ga5/modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L1857...L1873.